backend/src/modules/boardgame/boardgame.resolver.ts

```diff
@@ -1,14 +1,19 @@
-import { BoardgameInput } from './dto/boardgame.input';
-import { BoardgameOutput } from './model/boardgame.output';
+import { Args, Mutation, Parent, Query, ResolveProperty, Resolver } from '@nestjs/graphql';
+import { LanguageOutput } from '../language/domain/language.output';
 import { BoardgameService } from './boardgame.service';
-import { Resolver, Query, Args, Mutation } from '@nestjs/graphql';
+import { BoardgameInput } from './domain/boardgame.input';
+import { BoardgameOutput } from './domain/boardgame.output';

```

Comment:
The folder name "domain" is maybe not optimal, I may renamed it interfaces or IO